packages/pwa/app/utils/utils.js

```diff
@@ -164,7 +165,162 @@ export const capitalize = (text) => {
 export const getConfig = () => pwaKitConfig
 
 /**
- * A util to return current url configuration
- * @returns {object} - url object from the pwa-kit.config.json file
+ * Determine the site from the url
+ * @param {string} url
+ * @returns {object}
  */
-export const getUrlConfig = () => getConfig().url
+export const resolveSiteFromUrl = (url) => {
+    const {
+        app: {defaultSiteId, sites: sitesConfig}
+    } = getConfig()
+    let path = url
+    if (!path) {
+        path = `${window?.location.pathname}${window?.location.search}`
+    }
```

Comment:
It would make more sense to throw and error if the url isn't provided, this will make sure that this function is "universal" and be run on client and server.